openslides_backend/action/actions/meeting/update.py

```diff
@@ -32,6 +32,17 @@
     "conference_los_restriction",
     "conference_stream_url",
     "conference_stream_poster_url",
+    "conference_open_microphone",
+    "conference_open_video",
+    "conference_auto_connect_next_speakers",
+    "conference_enable_helpdesk",
+    "applause_enable",
```

Comment:
All these fields are not in the models.yml! Do not add them here until you changed the models.

Reply:
See the comment above from @jsangmeister